lib/matplotlib/colors.py

```diff
@@ -1722,6 +1720,11 @@ def __init__(self, boundaries, ncolors, clip=False, *, extend='neither'):
                              "number of bins")
 
     def __call__(self, value, clip=None):
+        """
+        This method behaves similarly to `.Normalize.__call__`, but returns
+        integers or arrays of int16, instead of floats in the ``[0, 1]``
```

Comment:
I don't think it needs the "instead of floats" parts. Folks can look up the other normalize methods if they need to know the difference, and that there is a difference has already been noted via the 'but'